src/generator/app.js

```diff
@@ -21,12 +21,20 @@ app.set('port', (process.env.PORT || 5000));
 
 // Use the www folder as static frontend
 app.use('/', express.static(__dirname + '/www'));
+app.use('/live/preview', express.static(__dirname + '/../../dist'));
+
+app.post('/live', uploadedFiles, function(req,res,next) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['next' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2758379534/issues/source?bid=3233857&fileBranchId=3359620#l26)